### DISCUSSION TITLE
Reading List Repository for the Weekly Research Group - incl for onboarding of new members

### DISCUSSION THREAD

**Discussion by baouroux** [life-itself/community#495] [General]
Picking up from last week's Weekly Research Group call, and following up on the ideas on collecting a 'shared knowledge' reference base, herewith a call for suggestions on the ways to take Matt's earlier Hackathon activity of Reading Lists, into a core and ongoing action for the Weekly Research Group...

Copying Matt's post from the "Research Community Hackathon / Create-athon (first one) #466" thrread: https://github.com/orgs/life-itself/discussions/466#discussioncomment-5744276 

**Reading lists / learning resource curation**
- A framework for organising learning resources for newcomers to a particular topic, i.e. books, papers, videos, podcasts, thinkers, etc. This could also be integrated with a knowledge map.
- Inspiration: [user-generated reading lists](https://placesjournal.org/reading-lists/) / [notion reading lists](https://parameter.io/notion-reading-list-templates/) / [roadmap.sh](https://roadmap.sh/system-design) / [healthygamer.gg](https://user-images.githubusercontent.com/960919/234788298-d2ef94a1-c7ae-4f7a-b108-19128f448cc1.png)

I take also a big inspiration from the way that @msantolini summarised the key texts on his journey over the years from [his presentation to our weekly group](https://docs.google.com/presentation/d/1t2fYeOA14PBu-TZ--yTYRdGyQNvqMYIry99K0GOU0c8/edit#slide=id.g217bf0a194c_0_148).

I would personally prefer the Notion platform, since I use Notion quite extensively!

**Comment by rufuspollock** [comment]
@baouroux to clarify: is this thread about the platform for curating these lists or for initial sharing of actual lists.

On the latter just came across a note in my journal about a first few books I'd recommend to 6th formers (i'm giving a talk to them soon):

- Island by Huxley - the classic (e)utopian novel
- Finders by Jeffery Martin - simple, no-nonsense intro to waking up with a securlar-ish presentation
- Mind Illuminated by Culadasa - core meditation manual with good scientific context
- Santaroga Barrier by Herbert - the neglected masterpiece. Take away the pot-boiler "monster in the lake" and you have the core of an counter-cultural ontological masterpiece (the hero is called Dasein for Heidegger fans)

**Reply by baouroux** [reply]
The post was intended to start the process in choosing a platform.
But why not start sharing initial lists, as you have! We can then always copy these across to the platform.
